doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -314,6 +317,10 @@ Documentation
   * Documentation for :ref:`degugging of nRF5340 <debugging>` in :ref:`working with nRF5340 DK<ug_nrf5340>` user guide.
   * Section describing how to enable Amazon Frustration-Free Setup (FFS) support in :ref:`ug_matter_configuring_device_identification`.
 
+* Updated:
+
+  * Improved :ref:`lib_nrf_cloud_rest` documentation, especially regarding connection timeout handling and error handling.
```

Comment:
I believe this does not need an entry here as it is already in the library changes.